src/libs/ReportUtils.ts

```diff
@@ -608,6 +608,13 @@ function isDraftReport(reportID: string | undefined): boolean {
     return !!draftReport;
 }
 
+/**
+ * Returns the report
+ */
+function getReport(reportID: string): OnyxEntry<Report> {
+    return ReportConnection.getAllReports()?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`];
+}
+
```

Comment:
FYI, I don't think we should be exporting getReport anymore. It was previously removed in https://github.com/Expensify/App/pull/43632. They added a test rule to prevent the export, but they got the name wrong.
https://github.com/Expensify/App/blob/7d58aa6694f8ef6eb298c35526a154f7e374fdd6/tests/actions/EnforceActionExportRestrictions.ts#L32-L35

Reply:
Thanks @bernhardoj.
To contrary, `ReportConnection.ts` is created to keep track of all the reports instead of subscribing it from all the components for the performance improvement in https://github.com/Expensify/App/pull/44068. Maybe we can move `getReport` there, though it is the same.
Thoughts @robertjchen ?